example/buttons.md

```diff
@@ -0,0 +1,9 @@
+# Adding buttons
+Use [.buttonBehavior( )](https://github.com/d3plus/d3plus-timeline#Timeline.buttonBehavior) to modify style of the timeline. Accepted values are "buttons" and "ticks". 
+
+```js
+new d3plus.Timeline()
+  .domain([2001, 2010])
+  .buttonBehavior("buttons")
```

Comment:
when the default value changes to "auto", the examples will now all show buttons. make this example show `"ticks"` instead